roles/addons/report/templates/hosts.css.j2

```diff
@@ -24,12 +24,15 @@ padding: 10px 10px;
 }
 
 {% for host in groups[rack] %}
-
+{% if hostvars[host]['x_p'] is defined and hostvars[host]['z_p'] is defined and hostvars[host]['x_span'] is defined and hostvars[host]['z_span'] is defined %}
```

Comment:
could you add some indentation for better readability? (will need the header `#jinja2: lstrip_blocks: True`)

Reply:
sure !